driver-examples/tycho/.gitignore

```diff
@@ -0,0 +1,2 @@
+!.classpath
```

Comment:
Oh, I did not realize that!  In that case I agree with keeping them in git then :+1: